docker/registry/cpp-client-base/gradle.properties

```diff
@@ -1,5 +1,3 @@
 io.deephaven.project.ProjectType=DOCKER_REGISTRY
 deephaven.registry.imageName=ghcr.io/deephaven/cpp-client-base:latest
-deephaven.registry.imageId=ghcr.io/deephaven/cpp-client-base@sha256:5b6e1749437170c0b0a3ebe5755c38fdbe3645b7c409cf8a72bc6a6acf50eebe
-# TODO(deephaven-base-images#54): arm64 native image for cpp-client-base
-deephaven.registry.platform=linux/amd64
```

Comment:
Sorry, this hasn't been incorporated into the update script. Please make sure this doesn't get removed.